app/controllers/projects_controller.rb

```diff
@@ -75,6 +75,7 @@ def allowed_includes
     [
       :environment_variable_groups,
       :environment_variables_with_scope,
+      (:external_environment_variable_groups if ExternalEnvironmentVariableGroup.configured?)
     ]
```

Comment:
having a nil in here could lead to trouble, better `.compact`
```suggestion
    ].compact
```

Reply:
this should be done in the `env` plugin via a decorator or a hook